apps/web/frontend/components/taskList/taskRow.tsx

```diff
@@ -37,27 +37,17 @@ export const TaskRow = ({ task: taskFragment }: TaskRowProps) => {
   const task = useFragment(TaskRowFragment, taskFragment)
   const [{ fetching: fetchingTitle }, updateTaskTitle] = useMutation(TaskUpdateMutationDocument)
   const [{ fetching: fetchingHourlyRate }, updateHourlyRate] = useMutation(TaskUpdateMutationDocument)
-  const {
-    setError,
-    register,
-    handleSubmit,
-    formState: { errors },
-  } = useForm<Pick<TaskUpdateInput, 'title'>>({
+  const { setError, register, handleSubmit, formState, reset } = useForm<TaskUpdateInput>({
     mode: 'onChange',
     defaultValues: {
       title: task.title,
-    },
-    resolver: zodResolver(taskInputValidations.pick({ title: true })),
-  })
-
-  const hourlyRateForm = useForm<Pick<TaskUpdateInput, 'hourlyRate'>>({
```

Comment:
I think we should keep two `useForm()` hook calls. Both inputs (task title and hourly rate) should work independently.

Reply:
I agree!